frontend/src/pages/docs/[...slug].tsx

```diff
@@ -179,8 +179,13 @@ const DirectoryListItem = ({
   directory: Directory;
   activeFile: string;
 }) => {
+  const initialState = useMemo(() => {
+    return directory.files.includes(activeFile) ? 1 : 0;
+  }, [directory.files, activeFile]);
+
```

Comment:
This is only looking at the current directory and its children, and opening if necessary.  There are also cases where the current directory contains additional directories, and inside of one of those directories is the active file. See: https://localhost:3000/docs/04__Analyze%20Public%20Data/4_2__Gene%20Expression%20Documentation/4_2_2__Cell%20Type%20and%20Gene%20Ordering

Reply:
just to make sure i'm understanding this correctly: so i had thought this line took care of that https://github.com/chanzuckerberg/single-cell-data-portal/blob/main/frontend/src/pages/docs/%5B...slug%5D.tsx#L226 but looking further, it looks like this will expand the "Gene Expression Documentation" directory, but not the "Analyze Public Data" directory, resulting in the navigation being collapsed since the "grandparent" directory is not expanded, even if the "parent" directory is?